report_thesis/src/sections/introduction.tex

```diff
@@ -5,30 +5,42 @@ \section{Introduction}\label{sec:introduction}
+To achieve these objectives, this research systematically explores a range of promising machine learning models and preprocessing techniques, identified through an extensive literature review and guided by a curiosity to explore unconventional approaches.
+Specifically, we implemented an experimental framework using an automated hyperparameter optimization tool to determine the most effective combinations of preprocessing methods and models for each major oxide analyzed.
+We began by evaluating various preprocessing techniques to understand their impact on model performance, selecting those that demonstrated the highest impact on improving the performance of each model.
+Following the preprocessing, the most promising models underwent an optimization process, allowing us to precisely tune both model configurations and their respective hyperparameters on a per-oxide basis, ensuring optimal performance tailored to the specific data characteristics of each oxide.
+Once the best configurations were identified, a stacking ensemble method was employed to create a meta learner for each oxide, significantly enhancing prediction accuracy and robustness beyond the capabilities of individual models.
+Through extensive experiments on \gls{libs} data, we systematically assessed and demonstrated the superior performance of our approach compared to existing methods, focusing on significant improvements in prediction accuracy and robustness.
 
 Our key contributions are as follows:
 \begin{itemize}
-    \item We develop a novel machine learning pipeline that effectively handles the challenges of \gls{libs} data to accurately predict major oxide compositions in Martian soil samples.
-    \item We conduct a comprehensive evaluation of various dimensionality reduction techniques and machine learning models to identify the optimal combination for this task. 
-	\item We demonstrate the superior performance of our approach compared to existing methods through extensive experiments on \gls{libs} data.
+    \item We develop a novel machine learning pipeline that effectively handles the challenges of LIBS data to accurately predict major oxide compositions in Martian soil samples.
+    \item We have developed a novel optimization approach and tool for tuning and evaluating machine learning models along with preprocessing techniques, providing a systematic and efficient method for selecting the best configuration.
+    \item Our investigations provides a methodological framework that advances the approach to model selection and tuning for high-dimensional, multicollinear geochemical data analysis.
+    \item Using this framework, we have identified and tuned the most promising models and preprocessing techniques from the literature for predicting major oxide compositions in Martian soil samples.
```

Comment:
This is something we did. It's not a contribution. It's an action.

Reply:
Do we remove it then? Or should it somehow be rephrased?

Reply:
Remove IMO